userspace/libsinsp/socket_handler.h

```diff
@@ -354,7 +358,7 @@ class socket_data_handler
 		ssize_t rec = 0;
 		std::vector<char> buf(1024, 0);
 		int counter = 0;
-		int processed = 0;
+		uint32_t processed = 0;
```

Comment:
Why is unsigned better here? The return value is `int` anyway.

Reply:
this is because `processed` is compared to `m_data_max_b`, which is uint32_t. Logically, in any case, it seemed more logical to have processed (which is a positive value) as an unsigned.